src/app/components/GuestTickets.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+import { useState } from 'react';
+import FormInput from '@/app/components/FormInput';
+import Link from 'next/link';
+
+export default function signup(props: any) {
+  const [guestData, setGuestData] = useState<{
```

Comment:
how will we use this i the rest of the application. See my other comment about what we will save to the database.

Reply:
The whole booking process is dynamically rendered so I don't think there is needed much logic to go to next step. Just update state and conditionally render the right things.